cli/src/cluster_query.rs

```diff
@@ -336,10 +342,10 @@ impl ClusterQuerySubCommands for App<'_, '_> {
                         "Only show stake accounts delegated to the provided vote accounts. "),
                 )
                 .arg(
-                    Arg::with_name("lamports")
-                        .long("lamports")
-                        .takes_value(false)
-                        .help("Display balance in lamports instead of SOL"),
+                    pubkey!(Arg::with_name("withdrawer_pubkey")
+                    .value_name("PUBKEY")
+                    .long("withdraw-autority"),
+                    "Filter stakes with the given withdraw authority. "),
```

Comment:
How about "Only show stake accounts with the provided withdraw authority. "?

https://github.com/solana-labs/solana/pull/25939#discussion_r897211030

Reply:
I think "Filter" is more technical than "Only show stake", but if more reviewers prefer later why not.